raiden/src/raiden.ts

```diff
@@ -375,11 +372,13 @@ export class Raiden {
    * @param address  Optional target address. If omitted, gets own balance
    * @returns  BigNumber containing address's token balance
    */
-  public async getTokenBalance(token: Address, address?: Address): Promise<BigNumber> {
+  public async getTokenBalance(token: string, address?: string): Promise<BigNumber> {
+    address = address || this.address;
+    if (!Address.is(address) || !Address.is(token)) throw new Error('Invalid address');
```

Comment:
nitpick: My only sligh objection here is that it is not too clear which address is invalid. Maybe in the future we can improve it to be more specific.

Reply:
It's not too much here to actually be used, the user is expected to handle a proper address every time. This line is just to ensure typescript knows the addresses are `Address` type/brand after this line. But yeah, I agree, this can improve later.